common/builder/pystache_message_builder.py

```diff
@@ -8,18 +9,18 @@
 logger = log.IntegrationAdaptorsLogger('COMMON_MSG_BUILDER')
 
 
-class PystacheMessageBuilder:
+class PystacheMessageBuilder(object):
     """A component that uses Pystache to populate a Mustache template in order to build a message."""
 
     def __init__(self, template_dir, template_file):
         """Create a new PystacheMessageBuilder that uses the specified template file.
         :param template_dir: The directory to load template files from
         :param template_file: The template file to populate with values.
         """
-        self._renderer = pystache.Renderer(search_dirs=template_dir, missing_tags=pystache_common.MissingTags.strict)
+        self._renderer = Renderer(search_dirs=template_dir, missing_tags=pystache_common.MissingTags.strict)
```

Comment:
I think this contradicts [rule 2.2 from the Google Python style guide](http://google.github.io/styleguide/pyguide.html#22-imports).